core/tests/protractor/userJourneys.js

```diff
@@ -15,20 +15,21 @@
 /**
  * @fileoverview End-to-end tests for user management.
  */
-
-var editor = require('../protractor_utils/editor.js');
-var forms = require('../protractor_utils/forms.js');
-var general = require('../protractor_utils/general.js');
-var users = require('../protractor_utils/users.js');
-var workflow = require('../protractor_utils/workflow.js');
-var LibraryPage = require('../protractor_utils/LibraryPage.js');
-
 var AdminPage = require('../protractor_utils/AdminPage.js');
+var CreatorDashboardPage =
+  require('../protractor_utils/CreatorDashboardPage.js');
+var CollectionEditorPage =
```

Comment:
Co... comes before Cr... alphabetically.

Reply:
Oops! Sorry, I'll be doing this. 
Thanks!

Reply:
Done!